Action.c

```diff
@@ -606,6 +606,7 @@ static const struct {
    { .key = "      u: ",  .roInactive = false, .info = "show processes of a single user" },
    { .key = "      H: ",  .roInactive = false, .info = "hide/show user process threads" },
    { .key = "      K: ",  .roInactive = false, .info = "hide/show kernel threads" },
+   { .key = "      O: ",  .roInactive = false, .info = "hide/show containerized processes" },
```

Comment:
How about "hide/show processes in containers" ?
Same length, seems more understandable / correct English.

Reply:
Looks good to me.